packages/smithy-http/src/smithy_http/aio/crt.py

```diff
@@ -187,7 +187,7 @@ async def send(
             return await self._await_response(crt_stream)
         except AwsCrtError as e:
             if e.name in self._TIMEOUT_ERROR_NAMES:
-                raise _CRTTimeoutError() from e
+                raise _CRTTimeoutError(f"CRT {e.name}: {e.message}")
```

Comment:
Ideally, we should be keeping the context with the `from` but we can include the message to make the immediate information clearer.